### PR TITLE
Add some Ddoc comments to dstruct.d

### PR DESCRIPTION
**Pull request by WalterBright**
Given https://dlang.org/phobos/ddmd_dstruct.html , I redid a few comment blocks to be more in Ddoc style.

### CODE REVIEW

File: src/ddmd/dstruct.d [hunk 1]
```diff
@@ -696,12 +706,16 @@ extern (C++) class StructDeclaration : AggregateDeclaration
...
-     * Return true if struct is POD (Plain Old Data).
+     * Determine if struct is POD (Plain Old Data).
+     *
      * This is defined as:
-     *      not nested
-     *      no postblits, destructors, or assignment operators
-     *      no 'ref' fields or fields that are themselves non-POD
+     *      1. not nested
+     *      2. no postblits, destructors, or assignment operators
+     *      3. no 'ref' fields or fields that are themselves non-POD
```
Comment (wilzbach) [inline src/ddmd/dstruct.d]: You probably want to use `$(UL)$(LI)` or line breaks here.

image

**Review by wilzbach — DISMISSED** [review]
Nice!
@CyberShadow's excellent DAutoTest also works with it:

http://dtest.dlang.io/artifact/website-40a2ba39d14b6f5f3d7bbcb3926711554d3a3311-121856aaf92414939d3f57ccf306d068/web/phobos-prerelease/ddmd_dstruct.html

(The Ddox files are hidden due to https://github.com/dlang/dmd/pull/6867)

File: src/ddmd/dstruct.d (hunk 1 above)
Comment (ZombineDev) [inline src/ddmd/dstruct.d]: There's also the option to implement https://issues.dlang.org/show_bug.cgi?id=16112 ;)

**Review by andralex — APPROVED** [review]
lgtm. @wilzbach ?

**Review by PetarKirov — APPROVED** [review]
LGTM too.

**Comment by ZombineDev** [comment]
BTW, 

> ... no `ref` fields...

Are `ref` fields really a thing?

**Review by andralex — APPROVED** [review]

**Comment by andralex** [comment]
@wilzbach I can't see what changes you requested. Clicking on "View changes" takes me nowhere interesting.

**Comment by ZombineDev** [comment]
@andralex Walter already addressed @wilzbach's request. This PR is good to go.

**Pull request merged**